smart_card_connector_app/src/chrome-api-jstocxxtest.js

```diff
@@ -97,6 +98,24 @@ goog.exportSymbol('testChromeApiProviderToCpp', {
         testController.executableModule.getMessageChannel(),
         pcscReadinessTracker.promise);
     await pcscReadinessTracker.promise;
+  },
+
+  'testEstablishContext': async function() {
+    chrome
+        .smartCardProviderPrivate['reportEstablishContextResult'](
+            /*requestId=*/ 123,
+            /*scardContext=*/ goog.testing.mockmatchers.isNumber, 'SUCCESS')
```

Comment:
That's good enough for the initial PR that introduces this test, but eventually it's probably better to double-check the received `scardContext` against `SCardIsValidContext()`.